tests/integration/ceph_base_object_test.go

```diff
@@ -192,18 +192,20 @@ func testObjectStoreOperations(s suite.Suite, helper *clients.TestClient, k8sh *
 	})
 
 	// Check object store status
-	t.Run("verify CephObjectStore status", func(t *testing.T) {
+	t.Run(fmt.Sprintf("verify ceph object store %q status", storeName), func(t *testing.T) {
 		i := 0
-		for i = 0; i < 10; i++ {
+		for i = 0; i < 30; i++ {
```

Comment:
How about defining a var for `30` so the assert below always uses the same value?